src/slicer.js

```diff
@@ -495,18 +494,18 @@
             });
         }
 
-        /* Send a index command to the Slicing Dice API
+        /* Send a insert command to the Slicing Dice API
          * 
          * @param (array) query - the query to send to Slicing Dice API
-         * @param (boolean) autoCreateFields - if is true Slicing Dice API will
-         * automatically create nonexistent fields
+         * @param (boolean) autoCreateColumns - if is true Slicing Dice API will
```

Comment:
"autoCreateColumns" parameter should be removed.
Check this PR: (https://github.com/SlicingDice/slicingdice-javascript/pull/12)